benchmarks/src/jmh/java/org/corfudb/benchmarks/runtime/collections/CorfuTableBenchmark.java

```diff
@@ -0,0 +1,175 @@
+    }
+
+    /**
+     * Put operation benchmark for RocksDb
+     *
+     * @param state     benchmark state
+     * @param blackhole jmh blackhole
+     */
+    @Benchmark
+    public void rocksDbPut(RocksDbStateForPut state, Blackhole blackhole) {
+        CorfuTableBenchmarkHelper helper = state.getHelper();
+        Integer key = helper.generate();
+        String result = helper.getUnderlyingMap().put(key, helper.generateValue());
+        blackhole.consume(result);
+    }
+
+    /**
+     * Put operation benchmark for EhCache
+     *
+     * @param state     benchmark state
+     * @param blackhole jmh blackhole
+     */
+    @Benchmark
+    public void ehCachePut(EhCacheStateForPut state, Blackhole blackhole) {
+        CorfuTableBenchmarkHelper helper = state.getHelper();
+        String result = helper.getTable().put(helper.generate(), helper.generateValue());
+        blackhole.consume(result);
+    }
+
+    /**
+     * Get operation benchmark for RocksDb
+     *
+     * @param state     benchmark state
+     * @param blackhole jmh blackhole
+     */
+    @Benchmark
+    public void rocksDbGet(RocksDbStateForGet state, Blackhole blackhole) {
+        CorfuTableBenchmarkHelper helper = state.getHelper();
+        int key = helper.generate();
+        String value = helper.getTable().get(key);
+
+        if (value == null) {
+            throw new IllegalStateException("The value not found in the cache. Key: " + key);
+        }
+
+        blackhole.consume(value);
+    }
+
+    /**
+     * Get operation benchmark for ehCache
+     *
+     * @param state     benchmark state
+     * @param blackhole jmh blackhole
+     */
+    @Benchmark
+    public void ehCacheGet(EhCacheStateForGet state, Blackhole blackhole) {
+        CorfuTableBenchmarkHelper helper = state.getHelper();
+        int key = helper.generate();
+        String value = helper.getTable().get(key);
+        if (value == null) {
+            throw new IllegalStateException("The value not found in the cache. Key: " + key);
+        }
+        blackhole.consume(value);
+    }
+
+    /**
+     * Put operation benchmark for HashMap
+     *
+     * @param state     benchmark state
+     * @param blackhole jmh blackhole
+     */
+    @Benchmark
+    public void hashMapPut(HashMapStateForPut state, Blackhole blackhole) {
+        CorfuTableBenchmarkHelper helper = state.getHelper();
+        String value = helper.getTable().put(helper.generate(), helper.generateValue());
+        blackhole.consume(value);
+    }
+
+    /**
+     * Get operation benchmark for HashMap
+     *
+     * @param state     benchmark state
+     * @param blackhole jmh blackhole
+     */
+    @Benchmark
+    public void hashMapGet(HashMapStateForGet state, Blackhole blackhole) {
+        CorfuTableBenchmarkHelper helper = state.getHelper();
+        String value = helper.getTable().get(helper.generate());
+        blackhole.consume(value);
+    }
+
+    /**
+     * Get/Put 50x50 load for HashMap
```

Comment:
renamed

Reply:
It will be nice to add updates and deletes too for complete fair comparison.

Reply:
1. Not sure how it can be measured without affecting the benchmark by `put` operation, you need to put some data and only then remove. The `update` operation is the same as `put`
2. What could be nice to measure is the speed of `compaction` algorithm, but it's for next pr-s